### PR TITLE
Phishing Detector, Fungible tokens support, Transactions bugfixing

### PR DESCRIPTION
**Pull request by petarbaykov**
- Fixing issue #2 
Waellet now detects phishing websites and warns you about it
- Fixing issue #15 
Waellet now supports adding of fungible tokens. Test cases for this functionality are added but more need to be and will be added
- Refactor transactions page to support naming related transactions

### CODE REVIEW

File: src/popup/router/pages/FungibleTokens.vue [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+                            this.token.precisionDisabled = false
+                            this.addStep = false
+                        });
+                });
+            });
+        },
+        searchTokenMetaInfo(address) {
+            this.loading = true
+            try {
+                this.sdk.contractCall(FUNGIBLE_TOKEN_CONTRACT,address,'meta_info')
```
Comment (petarbaykov) [inline src/popup/router/pages/FungibleTokens.vue:154]: @mradkov Can you check this calling of contract method meta_info. Although the function is public and do not change the state still takes AE for transaction. I tried setting gas and gasPrice to 0 but then the method cannot be called at all.

Comment (mradkov) [reply src/popup/router/pages/FungibleTokens.vue:154]: I’ll review it

**Review by mradkov — COMMENTED** [review]
I'm getting this error when trying to add token address.

image

**Comment by petarbaykov** [comment]
> I'm getting this error when trying to add token address.
> 
> image

Can you try with this address ct_2CC7FPsumg5azo7xNH6XS9wCnzFqVCdE4WrKzrUk6Z4jNeazF8

**Comment by mradkov** [comment]
> > I'm getting this error when trying to add token address.
> > image
> 
> Can you try with this address ct_2CC7FPsumg5azo7xNH6XS9wCnzFqVCdE4WrKzrUk6Z4jNeazF8

Okay, it works with this token address.

**Pull request merged**